### ISSUE TITLE
Musikcube does not play songs from one album

### ISSUE THREAD

**Issue by litearc** [clangen/musikcube#466]
Musikcube does not play songs from a specific album - when I try, it immediately says "playback is stopped". From the log file `~/.config/musikcube/log.txt`, the error is from ffmpegdecoder:
```
03:28:05 [info] [ffmpegdecoder] parsing data stream...
03:28:05 [error] [ffmpegdecoder] failed to find compatible audio stream
```
The album is "Chrono Trigger- Orchestral Selections Vol. I" (bought from Amazon). Interestingly, songs from Vol II play fine. Also, I can play the songs fine with `ffplay`. I don't know if this is a musikcube or ffmpeg issue.

I'm using Musikcube 0.96.6 #al7b9d91 (tried building from source also, issue persists), and on macOS Mojave.

**Comment by PythonTryHard** [comment]
Can you upload the file(s) to aid debugging?

**Comment by litearc** [comment]
Sure, here is one file from the album: https://drive.google.com/file/d/1RN7QR71PgL_KK4j1czGrAnDzX28wbe9i/view?usp=sharing
I tested by putting it in its own folder, adding it to my library, and trying to play it, which fails. Thanks.

**Comment by PythonTryHard** [comment]
This is...strange....I can't seem to replicate it, it plays just fine on my end without erroring out. What version of musikcube are you using, as well as OS?

**Comment by litearc** [comment]
I'm using Musikcube 0.96.6 `#al7b9d91` (tried building from source also, issue persists). I'm on macOS Mojave.
Edit: after a bit of digging, it seems `av_probe_input_format` in method `FfmpegDecoder::Open` is failing - not sure why though.

**Comment by PythonTryHard** [comment]
I don't have anything running macOS, but after some looking around I'll guess it's due to outdated `ffmpeg` and it's various libraries (`libavcodec`, the likes). So...maybe updating your `ffmpeg` could help....?

**Comment by litearc** [comment]
I have `ffmpeg 4.4_2`, which is the latest version. This is not really a big deal. It's weird it only happens on one album, and I'm guessing re-encoding the files will fix it.

**Comment by radluz** [comment]
I'm seeing the same behavior on several albums and I haven´t yet found a link.
Versions used:

> musikcube 0.99.5-1
ffmpeg 2:5.1.2-1
ffmpeg4.4 4.4.2-3

I have tried re-encoding the files, it worked for one album but not for the other. 
There's no available update for these packages that I can see. Very strange issue.

**Comment by clangen** [comment]
@radluz if you can attach a sample file I'll see if I can figure out what's going on.

**Comment by radluz** [comment]
Sorry for the delay I didn't have access to the files until now. Here are two of them:
https://drive.proton.me/urls/VSYTG9HFZM#sffGegNEiroW
https://drive.proton.me/urls/JRYJNZYXKG#n3980vJAG9uw

**Comment by clangen** [comment]
@radluz both of these files seem to play fine for me, are you running the most recent version of musikcube? The previous couple builds have had some decoder fixes that may address this problem.

If you _are_ using the most recent version, can you please tell me which platform you're running on, and if you compiled it yourself or installed a binary package?

**Comment by radluz** [comment]
I'm using v0.99.5-1 as described above. I'm running an Arch-based distro called [endeavourOS](https://endeavouros.com/) and the package was installed through using `yay`.
This is the one I have installed: https://aur.archlinux.org/packages/musikcube

**Comment by mostafaqanbaryan** [comment]
> I'm using v0.99.5-1 as described above. I'm running an Arch-based distro called [endeavourOS](https://endeavouros.com/) and the package was installed through using `yay`. This is the one I have installed: https://aur.archlinux.org/packages/musikcube

Hitting the same error on [endeavourOS](https://endeavouros.com/)

**Comment by henriquenunez** [comment]
Same problem here on musikcube 3.0.2_1, macos 14.4.1 (sonoma), ffmpeg 7.0_1, both installed via brew.
<img width="1185" alt="image" src="https://github.com/clangen/musikcube/assets/32100626/e87ef4f4-cb85-4f26-9f37-14bc76dade30">

**Comment by Reaper176** [comment]
same here.
Garuda --> Arch

have installed from: source, pacman, yay.

Has worked for me in the past, rather i should say it was working and than stopped working one day.

**Comment by Piker98988** [comment]
Same issue here on arch linux...

**Comment by henriquenunez** [comment]
Hey y'all, I solved it by deleting the `~/.config/musikcube` directory. It took me back to the initial setup, but fixed this dead state for me.

**Comment by Piker98988** [comment]
yeah, I personally changed music player... been using harmonoid recently so there it is.

**Comment by mokkun** [comment]
I'm having the same issue with some FLAC albums on the macOS. Other local players (i.e., QuickTime) can play the files.

Log error:
```
[I] [ffmpegdecoder] parsing data stream...
[e] [ffmpegdecoder] audio stream not found in input data.
[e] [ffmpegdecoder] failed to find compatible audio stream
[e] [ffmpegdecoder] open on, but decode failed /path/to/file.flac
```

You can download one of the files to verify the issue: https://d.pr/a/ckkh5V

Installed via homebrew.
```
~> musikcubed --version

  using lockfile at: /tmp/musikcubed.lock
  musikcubed version: 3.0.4 #4815178b
```
```
~> ffmpeg -version
ffmpeg version 7.0.2 Copyright (c) 2000-2024 the FFmpeg developers
built with Apple clang version 15.0.0 (clang-1500.3.9.4)
configuration: --prefix=/opt/homebrew/Cellar/ffmpeg/7.0.2_1 --enable-shared --enable-pthreads --enable-version3 --cc=clang --host-cflags= --host-ldflags='-Wl,-ld_classic' --enable-ffplay --enable-gnutls --enable-gpl --enable-libaom --enable-libaribb24 --enable-libbluray --enable-libdav1d --enable-libharfbuzz --enable-libjxl --enable-libmp3lame --enable-libopus --enable-librav1e --enable-librist --enable-librubberband --enable-libsnappy --enable-libsrt --enable-libssh --enable-libsvtav1 --enable-libtesseract --enable-libtheora --enable-libvidstab --enable-libvmaf --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libxvid --enable-lzma --enable-libfontconfig --enable-libfreetype --enable-frei0r --enable-libass --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libspeex --enable-libsoxr --enable-libzmq --enable-libzimg --disable-libjack --disable-indev=jack --enable-videotoolbox --enable-audiotoolbox --enable-neon
```
